pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -456,15 +456,9 @@ func Test_Run_OneVolumeAttachAndDetachMultipleNodesWithReadWriteOnce(t *testing.
 	nodesForVolume := asw.GetNodesForVolume(generatedVolumeName)
 
 	// check if multiattach is marked
-	// at least one volume should be marked with multiattach error
+	// at least one volume+node should be marked with multiattach error
 	nodeAttachedTo := nodesForVolume[0]
```

Comment:
how do we know this is the node that get the error?

Reply:
Because that function will return array of 1 element. There is no second element in this arrray. There were only 2 node+volumes added to reconciler. One was successfully attached and moved to actual_state_of_world and other was stuck in multiattach error.